Lab_1.ipynb

```diff
@@ -1,6 +1,700 @@
+      "text/plain": [
+       "<Figure size 1080x360 with 2 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "x = np.linspace(-3.5, 3.5, 1000)\n",
+    "y = stats.norm.pdf(x, loc = 0., scale = 1.)\n",
+    "\n",
+    "fig, ax = plt.subplots(1, 2, figsize=(15,5))\n",
+    "\n",
+    "ax[0].plot(x, y)\n",
+    "ax[0].tick_params(labelsize = 12)\n",
+    "ax[0].set_xlabel(\"Sigma σ\")\n",
+    "ax[0].set_ylabel(\"Probability Density\")\n",
+    "ax[0].set_title(\"Gaussian (Standard Normal) Distribution\")\n",
+    "\n",
+    "ax[1].semilogy(x, y)\n",
+    "ax[1].tick_params(labelsize = 12)\n",
+    "ax[1].set_xlabel(\"Sigma σ\")\n",
+    "ax[1].set_ylabel(\"Log(Probability Density)\")\n",
+    "ax[1].set_title(\"Semilog Gaussian (Standard Normal) Distribution\")\n",
+    "\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e3e02fc4-353f-4de5-8d54-36962abbf948",
+   "metadata": {},
+   "source": [
+    "There are built-in Python commands for various distributions that can be used to calculate the probability at a given sigma and the sigma from a given probability. To make sure these commands have been used correctly, their results can be compared to accepted values from a z-table. "
```

Comment:
Maybe explain that a z-table is just values tabulated from mean 0 and standard dev. 1 Gaussian that is used as a general reference.